### DISCUSSION TITLE
v1.5.0-rc.1 tag

### DISCUSSION THREAD

**Discussion by rphlo** [plausible/analytics#2454] [Self-Hosted Support] [answered]
Couldn't not notice the publication of v1.5.0-rc.1 tag.

Is it a official pre-release?
Any documentation upcoming?

**Comment by ruslandoga** [comment, marked as answer]
@rphlo 👋

Yes, I'll try it out on my instance and if everything's OK, I'll write up the changes and publish a new release candidate :)

**Reply by ruslandoga** [reply]
Published now: https://github.com/plausible/analytics/discussions/2460
